package.json

```diff
@@ -11,7 +11,8 @@
     "dev": "nodemon --exec babel-node src/index.js",
     "build": "babel src -d dist",
     "serve": "node dist/index.js",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "setup": "npm install --save-dev nodemon"
```

Comment:
When you do `npm install` for all npm packages, does it not install nodemon?

Reply:
that is exactly the problem.
It installes but i think its an older version or something along the lines. For some reasons the commands are not functional, and the setup reinstalls the babel packages

Reply:
it cant be an older version because package.json controls the versions. Lets look at it on Thursday and in the mean time I will try to re-create the issue myself

Reply:
well just clone the repository ant that should do it

Reply:
Well when I do:
1. `git clone git@github.com:rokaskasperavicius/csproject-api.git`
2. `cd csproject-api`
3. `npm install`
4. `npm run dev`
Everything works

Reply:
So I don't get why you need to install nodemon again.

Did you do `npm install` to install all packages after cloning the repository?

Reply:
yes. judging by the message that 3 packgaes were actually reinstalled it had done something. that made it work. Im not sure the command will be necesary for the teachers i just thought may be good to have a plan b ready